site/content/contribute/more-info/webapp/using-i18n-extract.md

```diff
@@ -13,7 +13,9 @@ aliases:
 This page demonstrates how to review your results and to verify if your extraction was successful or not. If the extraction was not successful this page also provides a workaround to correct for this.
 
 {{<note "NOTE:">}}
-These steps apply for the `mattermost-webapp` repository only.
+These steps haven't been updated since Mattermost has switched to using a {{<newtabref href="https://github.com/mattermost/mattermost-server" title="monorepo">}}. We're in the process of updating our I18n workflow and these corresponding docs.
```

Comment:
We're in the process of making more changes here, so I decided to just leave a disclaimer instead of changing this right now. I wonder how long this will live here before we actually make a change and update this page 😅